src/Disks/VolumeJBOD.cpp

```diff
@@ -40,20 +40,28 @@ VolumeJBOD::VolumeJBOD(
         auto ratio = config.getDouble(config_prefix + ".max_data_part_size_ratio");
         if (ratio < 0)
             throw Exception(ErrorCodes::EXCESSIVE_ELEMENT_IN_CONFIG, "'max_data_part_size_ratio' have to be not less then 0.");
+
         UInt64 sum_size = 0;
         std::vector<UInt64> sizes;
         for (const auto & disk : disks)
         {
-            sizes.push_back(disk->getTotalSpace());
-            sum_size += sizes.back();
```

Comment:
For example, the error was here: integer overflow and incorrect calculation. But I'm not sure what the effects were; therefore I don't mark it as a bugfix.